### PR TITLE
Fix the crash log without userid on SasquatchMacObjC app

### PR DESCRIPTION
**Pull request by ConnorQi**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

The setUserId function is called in the wrong order.

## Related PRs or issues

[#78109](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/78109)

### CODE REVIEW

File: SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift [hunk 1]
```diff
@@ -237,7 +237,7 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
       appCenter.setUserId(nil)
       break
     case .alertSecondButtonReturn:
-      let text = textView.string ?? ""
+      let text = textView.string
```
Comment (MatkovIvan) [inline SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift:240]: Looks like unrelated change

Comment (ConnorQi) [reply SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift:240]: Just to eliminate compilation warnings
